### ISSUE TITLE
Spread props to input component

### ISSUE THREAD

**Issue by ivan-cash** [hc-oss/react-tag-input-component#34]
Hi team, I really like the lib, and I would like to suggest you to spread the props coming from the `TagsInput` component to the input tag inside. Something like this:

<img width="368" alt="image" src="https://user-images.githubusercontent.com/103957571/196830656-850e7707-8a0d-4483-894a-ff871d552075.png">

Another workaround would be to create an additional prop in `TagsInput` component named `inputProps` or similar, in order to add props into `input` component. This solution would grant to add props like `onFocus`, `autoComplete`  or another prop that we use with testing purposes.

Thank you all.

**Comment by harshzalavadiya** [comment]
@ivan-cash good idea, should be easy to implement I will try to do pick it up in upcoming days.

but PRs are always welcome as well :)
